deployment/content-localization-on-aws.yaml

```diff
@@ -44,7 +50,7 @@ Resources:
     Properties:
       TemplateURL: !Join
         - ""
-        - - "https://solutions-reference.s3.amazonaws.com/aws-media-insights-engine/"
+        - - "https://solutions-reference.s3.amazonaws.com/media-insights-on-aws/"
```

Comment:
The template name for the new release got changed.
It used to be `media-insights-stack.template`
Now it is `media-insights-on-aws-stack.template`

check line 58